module3-crud-and-data-models/r1-schema-and-data-models/README.md

```diff
@@ -0,0 +1,219 @@
+## Introduction To Database Schemas
```

Comment:
Every reading should start with an H1 for the reading title, then it should have the summary and learning goals. Then each main topic is an H2, any further sub-topics or sections within them can be H3, H4 or a bold highlight.

These are there in the contributing guidelines in the repo README. You can also refer module 1 readings.